Starbucks/Controllers/CartViewController/CartViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  CartViewController.swift
+//  Starbucks
+//
+//  Created by Reuben Simphiwe Kuse on 2024/04/25.
+//
+
+import Foundation
+import UIKit
+
+class CartViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    
+    
+    // Dummy array of products in the cart
+    var cartProducts: [Product] = []
+    
+    
+    lazy var cartTableView: UITableView = {
+        let tableView = UITableView(frame: .zero, style: .plain)
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.register(ProductTableViewCell.self, forCellReuseIdentifier: "ProductTableViewCellID")
```

Comment:
Don't forget that you can use static variables for `ProductTableViewCellID` to avoid adding the string here.